lib/find_with_order/mysql_support.rb

```diff
@@ -18,8 +20,8 @@ def with_order(relation, column, ids, null_first: false)
       else
         column = column.to_s
       end
-      return relation.order("field(#{column}, #{ids.map(&:inspect).join(',')})") if null_first 
-      return relation.order("field(#{column}, #{ids.reverse.map(&:inspect).join(',')}) DESC")
+      return relation.order(sanitize_sql_for_order(["field(?, ?)", column, ids.map(&:inspect)])) if null_first
+      return relation.order(sanitize_sql_for_order(["field(?, ?)", column, ids.reverse.map(&:inspect)]))
```

Comment:
I think we can call this method directly.  Ex: `ActiveRecord::Base.sanitize_sql_for_order`. (Works in Rails 5.2)